Setup fails on `windows-latest`

Godot setup fails on `windows-latest`.

Snippet from action that reproduces the error:
```
jobs:
  reproduce-error:
    runs-on: windows-latest
    steps:
        - name: 🤖 Setup Godot binary
          uses: lihop/setup-godot@v2
          with:
            version: 4.3-stable
            bits: 64
            mono: false
```
Raw logs:
```
2025-01-06T05:01:54.1253586Z ##[group]Run lihop/setup-godot@v2
2025-01-06T05:01:54.1253921Z with:
2025-01-06T05:01:54.1254368Z   version: 4.3-stable
2025-01-06T05:01:54.1254630Z   bits: 64
2025-01-06T05:01:54.1254858Z   mono: false
2025-01-06T05:01:54.1255106Z   alias: godot
2025-01-06T05:01:54.1255311Z   cache: true
2025-01-06T05:01:54.1255557Z   export-templates: false
2025-01-06T05:01:54.1255981Z   github-token: ***
2025-01-06T05:01:54.1256246Z ##[endgroup]
2025-01-06T05:01:54.5441726Z 🪟 Windows detected
2025-01-06T05:01:54.5449876Z Detected version: v4.3-stable 64bit 
2025-01-06T05:01:54.5451811Z 📁 Creating install directory at D:\a\godot-glecs\godot-glecs/v4.3-stable
2025-01-06T05:01:54.5466530Z Trying to restore Godot download from cache...
2025-01-06T05:01:54.8282994Z Downloading Godot from https://github.com/godotengine/godot/releases/download/4.3-stable/Godot_v4.3-stable_win64.exe.zip
2025-01-06T05:01:55.3458303Z Caching Godot download...
2025-01-06T05:01:55.5098549Z [command]C:\Windows\System32\tar.exe --posix -z -cf cache.tgz -P -C D:/a/godot-glecs/godot-glecs --files-from manifest.txt
2025-01-06T05:01:59.6069587Z Cache Size: ~55 MB (57295740 B)
2025-01-06T05:01:59.6704729Z Cache saved successfully
2025-01-06T05:01:59.6730512Z Godot download cached.
2025-01-06T05:01:59.6738521Z 🗜 Extracting Godot archive
2025-01-06T05:01:59.7476515Z [command]"C:\Program Files\PowerShell\7\pwsh.exe" -NoLogo -NoProfile -NonInteractive -ExecutionPolicy Unrestricted -Command "$ErrorActionPreference = 'Stop' ; try { Add-Type -AssemblyName System.IO.Compression.ZipFile } catch { } ; try { [System.IO.Compression.ZipFile]::ExtractToDirectory('D:\a\godot-glecs\godot-glecs/v4.3-stable/Godot_v4.3-stable_win64.exe.zip', 'D:\a\godot-glecs\godot-glecs/v4.3-stable', $true) } catch { if (($_.Exception.GetType().FullName -eq 'System.Management.Automation.MethodException') -or ($_.Exception.GetType().FullName -eq 'System.Management.Automation.RuntimeException') ){ Expand-Archive -LiteralPath 'D:\a\godot-glecs\godot-glecs/v4.3-stable/Godot_v4.3-stable_win64.exe.zip' -DestinationPath 'D:\a\godot-glecs\godot-glecs/v4.3-stable' -Force } else { throw $_ } } ;"
2025-01-06T05:02:01.1707583Z Godot archive extracted to D:\a\godot-glecs\godot-glecs/v4.3-stable
2025-01-06T05:02:01.1708962Z 📁 Creating directory for binary at 'D:\a\godot-glecs\godot-glecs/v4.3-stable/bin'...
2025-01-06T05:02:01.1710381Z Directory 'D:\a\godot-glecs\godot-glecs/v4.3-stable/bin' created!
2025-01-06T05:02:01.1710967Z Adding binary directory 'D:\a\godot-glecs\godot-glecs/v4.3-stable/bin' to GITHUB_PATH...
2025-01-06T05:02:01.1716936Z Binary directory 'D:\a\godot-glecs\godot-glecs/v4.3-stable/bin' added to GITHUB_PATH!
2025-01-06T05:02:01.1718904Z 🔗 Creating symbolic link from 'D:\a\godot-glecs\godot-glecs/v4.3-stable/Godot_v4.3-stable_win64.exe' to 'D:\a\godot-glecs\godot-glecs/v4.3-stable/bin/godot'...
2025-01-06T05:02:01.1727128Z Symbolic link created!
2025-01-06T05:02:01.1727564Z Downloading mesa-dist-win...
2025-01-06T05:02:01.5244827Z ##[error]Error: Could not find a release for pal1000/mesa-dist-win (win32 x64)

```

--- a/lib/Windows.js
+++ b/lib/Windows.js
@@ -8,7 +8,7 @@ const glob = require("glob");
 const path = require("path");
 const retry = require("async-retry");
 
-const MESA_DIST_WIN_VERSION = "23.3.1";
+const MESA_DIST_WIN_VERSION = "24.3.2";
 const MONO_VERSION = "6.12.0.182";
 
 class Windows extends Platform {

